### DISCUSSION TITLE
Use charter as mathfont with lualatex

### DISCUSSION THREAD

**Discussion by schoeps** [tudace/tuda_latex_templates#402] [General LaTeX Questions]
I tried to match the mathfont with the text font using the mathdesign package (this was recommended somewhere in trustworthy sources of the www). Unfortunately, this kills other symbols in the text (ß and  ń for example). MWE attached. @TeXhackse: Suggestions?
```
%!TEX TS-program = lualatex
\documentclass[]{tudapub}
\usepackage[charter]{mathdesign}
\begin{document}
12345 and $12345$

Straße

Spanish letter: ń

\end{document}

```

**Comment by TeXhackse** [comment]
Mathdesign is automatically loading T1 encoding. Therefore it should not be used with lualatex.

You can setup the available symbols using unicode math but charter simply lacks a lot of glyphs there.

**Reply by schoeps** [reply]
This makes sense. I understand the problem now but what can I do instead? :)

**Reply by TeXhackse** [reply]
Create a math font for charter? ;-) 

You can have a look at https://ctan.org/pkg/xcharter-math I did not have time to check that one out in detail and it's still beta.

**Reply by schoeps** [reply]
> Create a math font for charter? ;-)

Great, this is exactly the (short-term) solution that I was looking for :) 
Seriously, yes `[xcharter-math](https://ctan.org/pkg/xcharter-math)' seems to do the trick. Thanks. 

MWE:
```
%!TEX TS-program = lualatex
\documentclass[]{tudapub}
\usepackage{unicode-math}
\setmathfont{XCharter Math} 
\begin{document}
12345 and $12345$

Straße

Spanish letter: ń

\end{document}

```
